xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd

```diff
@@ -457,12 +458,81 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 			<xsd:element name="Portable" type="xsd:boolean" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Whether vehicle can be carried easily, e.g. scooter, skateboard, collapsible bicycle.
-</xsd:documentation>
+					<xsd:documentation>Whether vehicle can be carried easily, e.g. scooter, skateboard, collapsible bicycle.</xsd:documentation>
```

Comment:
```suggestion
					<xsd:documentation>Whether vehicle can be carried easily, e.g. scooter, skateboard, foldable bicycle.</xsd:documentation>
```

Reply:
Perhaps it's only because I'm more familiar with bicycles than with the English language - I'd clearly prefer "foldable" or "folding" to "collapsible" :-)